utils/token_tracker.py

```
@@ -71,32 +76,164 @@ def _deep_copy_day(day: dict) -> dict:
+
+# ---------------------------------------------------------------------------
+# 远程遥测上报配置（参考 vLLM DO_NOT_TRACK 机制）
+#
+# 设计与 vLLM 一致：秘钥和地址硬编码在源码中，无需用户配置环境变量。
+# HMAC 不是为了防止逆向（代码本身可读），而是防止随机噪声和简单伪造。
+# ---------------------------------------------------------------------------
+
+# ★ 发版前修改：遥测服务器地址。为空则不上报。
+_TELEMETRY_SERVER_URL = "http://project-neko.cn:8099"
+
+# ★ 发版前修改：HMAC 签名密钥（与 server.py 中的 HMAC_SECRET 保持一致）
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)